doc/source/Interface/ControlsAndShortcuts/index.md

```diff
@@ -257,3 +296,21 @@ Adjust frame(s) of selected key(s)              :kbd:`Shift` + click and drag le
 Adjust value(s) of selected key(s)              :kbd:`Shift` + click and drag up or down
 =============================================== =============================================
 ```
+
+
+## Plug Values ##
+
+> Note :
```

Comment:
There are a few non-numeric shortcuts we could include here, that work on file path widgets and also the same widgets when used to select a location in the scene :

- Tab to autocomplete
- Double click to select single component, and show a menu of files/locations at that level
- Triple click to select whole path, and show menu of all parent directories/locations

Reply:
Will add. Had no idea about the path components!

Reply:
Yeah, I'm rather fond of the path component interaction - all the benefits of text and tab complete, but with the same functionality you'd get from the series of dropdown menus that folks seem to be used to. The problem is discoverability...